multipledropdown/doc/ReadMe.md

```diff
@@ -0,0 +1,29 @@
+**MultipleDropDown**
+
+MultipleDropDown custom widget is used to visualize the data as in the below image.
```

Comment:
Have proper content. we have already prepared the use cases. get the proper content. this looks useless.

Reply:
Corrected the MD file content as per the suggestion.